plenum/server/message_req_processor.py

```
@@ -82,7 +82,7 @@ def _validate_requested_ledger_status(self, **kwargs):
                     # the input is expected as a dict (serialization with ujson==1.33)
                     return LedgerStatus(**kwargs['ledger_status'])
                 except TypeError as ex:
-                    logger.warning(
```

Let's not hide this exception. I think warning or error should be fine